frontend/src/features/mapSideBar/SidebarContents/BuildingForm.tsx

```diff
@@ -74,14 +176,31 @@ const BuidingForm = () => {
           persistCallback: noop,
         }}
         // provide initial building props
-        initialValues={{
-          activeStep: 0,
-          data: { name: 'Building name' },
-        }}
+        initialValues={initialValues}
+        formikRef={formikRef}
         // Provide onSubmit
-        onSubmit={values => alert(JSON.stringify(values.data))}
+        onSubmit={async (values, actions) => {
+          const apiValues = valuesToApiFormat(_.cloneDeep(values));
+
+          //temporary until we support buildings with no parcel
+          apiValues.latitude = apiValues.buildings[0].latitude;
```

Comment:
technically I think we already do, although some further api updates may be required to get everything 100% working.

Reply:
The db does, but the api continues to expect the relationship.